cli/command/init/cmd.go

```diff
@@ -109,3 +138,50 @@ func Run(cmd *cobra.Command, cli *command.SauceCtlCli, args []string) error {
 	return nil
 }
 
+func batchMode(cmd *cobra.Command, initCfg *initConfig) error {
+	stdio := terminal.Stdio{In: os.Stdin, Out: os.Stdout, Err: os.Stderr}
+	// TODO: Implement logic for using cfg.username / cfg.accessKey
```

Comment:
Are we planning to do that in the future? 🤔

Reply:
This is a remanent comment. I'm not sure that is making sens as in batchMode we should use configure or env vars